packages/portalnetwork/EXAMPLES.md

```diff
@@ -0,0 +1,50 @@
+# Starting a Portal Network Client
```

Comment:
This is a really interesting idea but we need to have the example script in `portalnetwork` for this to make sense.  

So, for starters, let's move your `portalClient.ts` script from `packages/cli/scripts` to a new `packages/portalnetwork/examples` directory.

Second, since your code is instantiating an `UltralightProvider` instead of a pure portal network client, can you invoke the `provider.request` method instead of accessing the `provider.portal` APIs?  

Otherwise, can you rewrite the script so it instantiates a pure Portal Network client via `PortalNetwork.create` instead of the provider?